agent/request.go

```diff
@@ -201,16 +208,34 @@ func (e *HTTPRequestEvent) GetName() string {
 	return e.method
 }
 
-func (e *HTTPRequestEvent) GetArgs() api.ListValue {
-	opts := api.NewRequestRecord_Observed_SDKEvent_OptionsFromFace(e)
-	return api.ListValue([]interface{}{e.event, opts})
+func (e *HTTPRequestEvent) GetEvent() string {
+	return e.event
```

Comment:
I only found 1 usage of this one, seems a bit overkill for now

Reply:
Usage of `GetEvent()` ? If so, these getters are related to the serializatio to the JSON format: the structures that will be serialized into JSON are created from types implementing their interface. So for example, here, this method is required by https://github.com/sqreen/go-agent/blob/f2a2478d638b6212821731701a6b4b7c6a99a2a3/agent/backend/api/api.pb.go#L2194 to create the track event arguments out of this structure.

Let me know if it's unclear.

Reply:
Mostly: I found only 1 use of a method which body is `return e.event` which could be inlined in usage easely